appshell/mac/Brackets.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Give preference to launching the one in /Applications folder
```

Comment:
In particular, this also means that the result of running "Brackets" from the command line may not be to run the copy of Brackets where the "install command line" menu item was run -- it could be a totally different copy instead.  That seems confusing.